README.md

```diff
@@ -1,51 +1,59 @@
 candi (Compile &amp; Install)
 =====
 
-This shell script downloads, configures, builds, and installs deal.II with common dependencies on linux-based systems.
+The `candi.sh` shell script downloads, configures, builds, and installs deal.II
+with common dependencies on linux-based systems.
 
 Quickstart
 ----
 
-The following commands download the current version of the installer and then installs
-the latest deal.II release and common dependencies:
+The following commands download the current version of the installer and
+then installs the latest deal.II release and common dependencies:
 
 ```bash
  git clone https://github.com/dealii/candi
  cd candi
- ./candi.sh -j2
+ ./candi.sh
 ```
 
-Follow the instructions on the screen (you can abort the process by pressing < CTRL > + C)
+Follow the instructions on the screen
+(you can abort the process by pressing < CTRL > + C)
 
 ### Examples
 
-#### Install deal.II on RHEL 7, CentOS 7 or Fedora 21/22:
```

Comment:
can we remove all the ``export ....`` lines because the compilers are detected automatically?